### PR TITLE
Make things work with python3

### PR DESCRIPTION
**Pull request by sean-lockwood**
~Results need to be checked!~

Also update installation instructions.

### CODE REVIEW

**Comment by sean-lockwood** [comment]
@ariley1472 
I got `basedark` and `weekdark` working in a fresh Astroconda python=3.7 environment, but we should validate the results against delivered reference files.

Also, the bias routines need to be tested.

This has become urgent because python2 environments are now breaking, making installation difficult for `refstis` and impossible for the `stis_cti` package that depends on it.

File: docs/conf.py [hunk 1]
```diff
@@ -52,17 +52,17 @@
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.6.1'
+version = '0.7.0'
 # The full version, including alpha/beta/rc tags.
-release = '0.6.1'
+release = '0.7.0a1'
```
Comment (sean-lockwood) [inline docs/conf.py]: This will need to be updated to `0.7.0` before merging.

**Comment by sean-lockwood** [comment]
@ariley1472 --
I think we're ready for a final review now that this has been scientifically vetted.

To review the docs:
```
cd docs
make clean html
cd _build/html
python3 -m http.server
```

**Comment by sean-lockwood** [comment]
Some testing results
===============

We used a combination of this new python3 `refstis` and [`run_refstis()`](https://grit.stsci.edu/lockwood/run_refstis) on grit to produce new super-biases and super-darks.  We expect the input files not to match exactly, but the scientific validity of the reference files appears to be ok.

**Bias gain=1:**
[check_crds_biases_with_new_2020Mar06.pdf](https://github.com/spacetelescope/refstis/files/4417595/check_crds_biases_with_new_2020Mar06.pdf)

**Bias gain=4:**
[check_crds_biases_gain4.pdf](https://github.com/spacetelescope/refstis/files/4417599/check_crds_biases_gain4.pdf)

**Darks:**
stis_superdark_stats
Zoomed in:
stis_superdark_stats_zoomed

Latest dark delivery (not yet sent to CRDS):
[stis_ccd_ref_comparison.pdf](https://github.com/spacetelescope/refstis/files/4417639/stis_ccd_ref_comparison.pdf)

File: docs/installation.rst [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
+Install via Anaconda
+--------------------
 
-  This package has a lingering dependency on an IRAF task that is accessed
-  through PyRAF.  IRAF/PyRAF can be obtained through
-  `Astroconda <http://astroconda.readthedocs.io/en/latest/>`_.  Though many
-  tasks will work without IRAF installed, some will fail under certain
-  circumstances.  This depencency will be removed in a future release as
-  resources permit.
+TBD.
```
Comment (dougbrn) [inline docs/installation.rst:7]: @sean-lockwood just want to confirm that the intent is to have these instructions not present in this release?

Comment (sean-lockwood) [reply docs/installation.rst:7]: I'm going to work on the conda installation after the merge.  Will update the instructions accordingly.

**Comment by sean-lockwood** [comment]
Thanks, @dougbrn!

The 2019 dark differences look to be due to a different number of input files due to USEAFTER date changes.  The factor of 2 in the bias issue indirectly affects some of these (by date), but it likely only has a very minor impact on the super-darks.

It does seem strange that the mean stays roughly the same and the median jumps.

**Pull request merged**